src/util/sortData.js

```diff
@@ -0,0 +1,23 @@
+export function sort(collection, attribute, reversed) {
+  if (!attribute) {
+    if (reversed) {
+      return collection.reverse();
+    } else {
+      return collection;
+    }
+  }
+
+  const col = collection.sort((a, b) => {
+    if (attribute[0] !== '#') {
```

Comment:
We willen dit niet op basis van het `#` teken doen, deze tekst kan altijd veranderen en dan werkt de sort niet meer.

Het is beter om per kolom aan te geven of het alphabetisch of numeriek gesorteerd moet worden.